readthedocs/organizations/templates/organizations/organization_choose.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load gravatar %}
+{% load organizations %}
+
+{% block title %}{% trans "Choose an organization" %}{% endblock %}
+
+{% block organization-bar-organization %}active{% endblock %}
+
+{% block content %}
+  <div class="col-major organization-major">
+    <div class="module organizations organizations-list">
+      <div class="module-wrapper">
+
+        <h2>{% trans "Before continuing, choose an organization..." %}</h2>
+
+        <p>
+          {% trans "You are a member of several organizations." %}<br>
+          {% trans "Select the organization that you want to use:" %}
+        </p>
+
+          {% if organization_list %}
+            <!-- BEGIN organization list -->
+            <div class="module-list">
+              <div class="module-list-wrapper">
+
+                <ul>
+                  {% for org in organization_list %}
+                    <li>
+                      <a href="{% url next_name slug=org.slug %}{% if next_querystring %}?{{ next_querystring }}{% endif %}">
```

Comment:
@agjohnson I'm not sure if this went unnoticed? But by parameterizing the chooser page with the url name of the page we want to redirect to, we can generate all the individual links using `next_name` and `org.slug`.

Reply:
Hrm, not sure I follow, is there an issue with this approach? I think using `url` like this was I think the intention with this view from the underlying discussion. If there was a question here, I think this looks good :+1:

Reply:
If you are hand-crafting a `/redirect/?next=view_name`, wouldn't you find it cumbersome to have to have knowledge about the Django urlconf's view name? And if you think about alternatives, aren't they just as annoying (manipulating URL paths either manually or with some automated logic to inject slugs)?

Reply:
I would not find this cumbersome as we are not talking about doing this frequently. We are only discussing doing this a few times, for docs and support templates.

I feel like I'm looping though. I think @ericholscher and I have both picked up on the same issue with this PR and without the decorator change this PR would be about ready to merge.

Reply:
I don't think that's a wise choice. You throw away a perfect generic solution that will immediately solve the practicality of "how to create a correct redirect link".

It can also be used for many other purposes, such as adapting it to "the organization does not exist or you don't have access to this organization, please choose from the list of organizations that you are a member of"

Reply:
Note that I wasn't trying to scope creep this, I was merely trying to solve it in a way that I believe is most feasible for the immediate use case.

Reply:
I removed the feature. I want to move on from this discussion. I think it was a nice feature but I'm not willing to defend its usefulness more than what I already did. Seems there's a "popular" demand to not have it so that's fine :+1: